### DISCUSSION TITLE
If one has a choice, is there a store that works better with Heroic than another, or no difference?

### DISCUSSION THREAD

**Discussion by Sunspark-007** [Heroic-Games-Launcher/HeroicGamesLauncher#3959] [General]
Some games are on Amazon, GOG and Epic and others.. I've wondered, does any given platform provide them in a way that works better with Heroic than another, or it makes no difference?

**Comment by DumbAsFuck** [comment]
gog will

**Comment by arrowgent** [comment]
if youre asking which store to use

amazon has game
gog has game
epic has game

then i dont think it will matter.
given a choice, use gog because it is DRM free.
